### PR TITLE
Update environment-set-up.md - Add version and keywords to default output of `npm init`

### PR DESCRIPTION
**Pull request by ElijahLynn**
**Description**:
re: https://docs.hedera.com/hedera/getting-started/environment-set-up

Super nit picky, but it is important to keep docs up-to-date so they don't drift, so :shrug:.

This PR updates the example of an `npm init` output to include a default version of 1.0.0 and an empty `keywords` array. The current example shows an empty string. https://docs.hedera.com/hedera/getting-started/environment-set-up

Current:
image

Actual:
image

### CODE REVIEW

File: getting-started/environment-set-up.md [hunk 1]
```diff
@@ -481,5 +482,7 @@ func main() {
 
 
 {% hint style="info" %}
-Have a question? [Ask it on StackOverflow](https://stackoverflow.com/questions/tagged/hedera-hashgraph)
+Have a question? [Ask it on StackOverflow  "version": "",
+57
+](https://stackoverflow.com/questions/tagged/hedera-hashgraph)
```
Comment (ElijahLynn) [inline getting-started/environment-set-up.md]:
```suggestion
Have a question? [Ask it on StackOverflow](https://stackoverflow.com/questions/tagged/hedera-hashgraph)
```

**Comment by theekrystallee** [comment]
Thanks for catching this. Please include a "Signed-off-by" line in the commit message.

To avoid having PRs blocked in the future, always include Signed-off-by: Author Name <authoremail@example.com> in every commit message. You can also do this automatically by using the -s flag (i.e., git commit -s).

**Comment by ElijahLynn** [comment]
> 

Thanks @theekrystallee! I just configured 1password, GitHub and [git](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) to sign with my SSH key and force pushed, and I'm seeing them as all verified now!

**Comment by ElijahLynn** [comment]
Odd though, the DCO check isn't passing though, I wonder if I should push a new commit.

**Comment by ElijahLynn** [comment]
I just read https://github.com/apps/dco and I get it now, `-S` is `--gpg-sign` and `-s is --signoff`, I did both on a rebase of my last 3 commits and force pushed now and it is good!

e.g.
`git rebase --exec 'git commit --amend --no-edit --no-verify --gpg-sign --signoff' HEAD~3`

**Review by theekrystallee — APPROVED** [review]
Thanks for the PR and again for catching this!

**Pull request merged**